streamly-coreutils.cabal

```diff
@@ -85,7 +85,8 @@ library
   build-depends:
       base >= 4.8 && < 5
     , exceptions >= 0.8  && < 0.11
-    , streamly >= 0.9.0 && < 0.10
+    -- We use internal modules of streamly.
+    , streamly == 0.9.0.*
```

Comment:
Isn't it the same?

Reply:
No, I don't want to allow all minor versions. Only patch versions for now.